martian/src/stage.rs

```diff
@@ -125,7 +125,7 @@ pub struct Resource {
     threads: Option<isize>,
     #[serde(rename = "__vmem_gb")]
     vmem_gb: Option<isize>,
-    #[serde(rename = "__special")]
+    #[serde(rename = "__special", skip_serializing_if = "Option::is_none")]
```

Comment:
Really this should probably be done for the other resource requests as well.

Reply:
Done